File: frontend2/src/app/shared/models/order.model.ts
import { OrderProduct } from './order-product.model';
import { PaymentMethod } from '../enums/payment-method.enum';
import { Customer } from './customer.model';

export class Order {
  id: string;
  orderDate: string;
  number: string;
  cost: number;
  costMethod: PaymentMethod;
  discount: number;
  profit: number;
  region: string;
  city: string;
  notes: string;
  isWholesale: boolean;
  products: OrderProduct[];
  customer: Customer;

  constructor(data: any) {
    Object.assign(this, data);
  }
}

File: Mushka.WebApi/ClientModels/Order/GetById/OrderModel.cs
using System;
using Mushka.Domain.Entities;

namespace Mushka.WebApi.ClientModels.Order.GetById
{
    public class OrderModel
    {
        public Guid Id { get; set; }

        public DateTime OrderDate { get; set; }

        public string Number { get; set; }

        public decimal Cost { get; set; }

        public PaymentMethod CostMethod { get; set; }

        public int Discount { get; set; }

        public decimal Profit { get; set; }

        public string Region { get; set; }

        public string City { get; set; }

        public bool IsWholesale { get; set; }

        public string Notes { get; set; }
    }
}